examples/build/README.md

```diff
@@ -0,0 +1,35 @@
+# Build operations
+
+This example program demonstrates the fundamental operations for working
+with
+[Build](https://docs.openshift.com/container-platform/3.7/dev_guide/builds/basic_build_operations.html)
+resources, such as `List builds`, `Retrieve details of a build` and `Trigger a build`.
+
+## Running this example
+
+Make sure you have an OpenShift cluster and `oc` is configured:
+
+```sh
+$ oc get nodes
+### Create a project
+$ oc new-project testproject
+### Create an app
+$ oc new-app https://github.com/openshift/cakephp-ex
+$ oc get build
+NAME           TYPE      FROM          STATUS     STARTED          DURATION
+cakephp-ex-1   Source    Git@e04b8cc   Complete   37 minutes ago   1m39s
+```
+
+Compile this example on your workstation:
+
+```sh
+### Load dependencies if not done yet (required only once)
+$ glide up -v
```

Comment:
This surprises me.  I was fairly sure we include everything in our vendor tree already.

Reply:
If we do not do the command, we would get this when building:

[hongkliu@hongkliu client-go]$ go build -o examples/build/app ./examples/build/
vendor/k8s.io/client-go/tools/clientcmd/auth_loaders.go:26:2: cannot find package "github.com/howeyc/gopass" in any of:
        /home/hongkliu/repo/go/src/github.com/openshift/client-go/vendor/github.com/howeyc/gopass (vendor tree)
        /home/hongkliu/tool/go/src/github.com/howeyc/gopass (from $GOROOT)
        /home/hongkliu/repo/go/src/github.com/howeyc/gopass (from $GOPATH)
vendor/k8s.io/client-go/tools/clientcmd/client_config.go:28:2: cannot find package "github.com/imdario/mergo" in any of:
        /home/hongkliu/repo/go/src/github.com/openshift/client-go/vendor/github.com/imdario/mergo (vendor tree)
        /home/hongkliu/tool/go/src/github.com/imdario/mergo (from $GOROOT)
        /home/hongkliu/repo/go/src/github.com/imdario/mergo (from $GOPATH)